### PR TITLE
Add back manual install content to mobile docs

### PR DESCRIPTION
**Pull request by ghost**
Needed to add back manual previously removed install content for native and hybrid mobile monitoring agents. I also improved some of the content organization to match the other mobile agent docs.

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Manually install the Android agent (Gradle/Android Studio)
+title: Manually install the Android agent (Android Studio/Gradle)
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx:2]:
```suggestion
title: Manually install the Android agent (Gradle/Android Studio)
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx [hunk 2]
```diff
@@ -35,9 +35,8 @@ redirects:
   - /docs/mobile-monitoring/new-relic-mobile-android/install-configure
 ---
 
-Our [guided install](https://onenr.io/0kjnEoryzRo) is the preferred method of installation, but if you need to manually install and configure our Android agent, follow the instructions below.
 
-If you haven't already, create your free New Relic account below to start monitoring your data today.
+We recommend that you use our [guided install](https://onenr.io/0kjnEoryzRo) to set up Android monitoring. However, if you need to manually install for any reason, follow the steps below to install the New Relic Android agent with Gradle and Android Studio.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx:39]: I would remove the "for any reason" as it seems unneeded, but it's just a small thing to consider.

File: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx [hunk 3]
```diff
@@ -14,7 +14,7 @@ import mobileHybridSample from 'images/mobile_screenshot-full_hybrid-summary.web
 
 import mobileQueryExample from 'images/mobile_screenshot-full_query-example.webp'
 
-Our New Relic Cordova agent monitors your Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the Cordova agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:
+Our New Relic Cordova agent monitors your Cordova mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the Cordova agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx:17]: Is there a reason to mention the Javascript? If not I think we can cut it safely.

File: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx [hunk 4]
```diff
@@ -25,13 +25,99 @@ Our New Relic Cordova agent monitors your Capacitor mobile app and provides deep
...
 To install the Cordova agent, follow our [guided install](https://onenr.io/08jqaJD9pjl), located directly in the UI.
 
+If you need to manually install for any reason, follow these steps
+
+<Steps>
+  <Step>
+
+### Create an application token(s) [#app-token]
+
+  1. Go to **one.newrelic.com > Browser & Mobile > (choose Android or iOS)**, then complete the instrumentation steps in the UI.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx:45]: I think there's supposed to be a link to the platform here

File: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx [hunk 5]
```diff
@@ -25,13 +25,99 @@ Our New Relic Cordova agent monitors your Capacitor mobile app and provides deep
...
+
+</Step>
+
+<Step>
+
+### (Optional) Configure the agent [#configure-agent]
+
+To add optional configuration options, add them to the `--variable` argument, which will call them upon agent start. 
+
+The agent supports these configuration options:
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx:67]: I think you can join this sentence with the previous paragraph

File: src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx [hunk 6]
```diff
@@ -30,19 +30,119 @@ Our New Relic Flutter agent monitors your Flutter mobile app and provides deep i
...
-* Flutter 2.5.0 or higher
-* Dart versions 2.16.2 or higher, up to but not including 3.0.0
-* [Android-native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
-* [iOS-native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
 
 ## Install the Flutter agent [#install]
 
 To install the Flutter agent, follow our [guided install](https://onenr.io/0gR70k5oZjo), located directly in the UI.
 
+If you need to manually install for any reason, follow these steps:
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx:38]: I think the for any reason can be cut here as well

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 7]
```diff
@@ -1,35 +1,167 @@
...
-metaDescription: "New Relic's Capacitor agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+metaDescription: "New Relic's Ionic Capacitor agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
 redirects:
   - /docs/mobile-monitoring/new-relic-mobile-capacitor/get-started
 ---
 
 import mobileQueryExample from 'images/mobile_screenshot-full_query-example.webp'
 
-Our New Relic Capacitor agent monitors your Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the plugin automatically includes New Relic's native agents to provide mobile monitoring and performance visibility. Once you install and configure the Capacitor agent, you'll be able to:
+Our New Relic Ionic Capacitor agent monitors your Ionic Capacitor mobile app and provides deep insights into your app's performance, errors, and user experience. Written in JavaScript, the plugin automatically includes New Relic's native agents to provide mobile monitoring and performance visibility. Once you install and configure the agent, you'll be able to:
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:14]: Again, if there's no reason to mention JavaScript then we should cut that part, but if there is we should keep it.

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 8]
```diff
@@ -1,35 +1,167 @@
...
 * [iOS-native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
 
-## Install [#install]
-
-To install the Capacitor agent, follow our [guided install](https://onenr.io/0VRVoxJW1ja), located directly in the UI.
+## Install the agent [#install]
+
+To install the agent, follow our [guided install](https://onenr.io/0VRVoxJW1ja), located directly in the UI.
+
+If you need to manually install for any reason, follow these steps:
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:34]: I think we can cut "for any reason" and join the sentence with the previous line.

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 9]
```diff
@@ -1,35 +1,167 @@
...
+
+<Steps>
+  <Step>
+
+### Create an application token(s) [#app-token]
+
+  1. Go to **one.newrelic.com > Browser & Mobile > (choose Android or iOS)**, then complete the instrumentation steps in the UI.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:41]: I think this is supposed to link into the platform in some way

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 10]
```diff
@@ -1,35 +1,167 @@
...
+      }
+    }
+  ```
+3. Update `app/build.gradle`:
+  ```groovy
+    apply plugin: "com.android.application"
+    apply plugin: 'newrelic' // <-- add this
+  
+  ```
+4. Make sure your app requests INTERNET and ACCESS_NETWORK_STATE permissions by adding these lines to your `AndroidManifest.xml`:
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:142]: Should INTERNET and ACCESS_NETWORK_STATE be in code format?

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 11]
```diff
@@ -10,10 +10,9 @@ redirects:
   - /docs/mobile-monitoring/mobile-monitoring-installation/ios/cocoapods-installation-and-configuration
   - /docs/mobile-monitoring/mobile-monitoring-installation/ios/cocoapods-installation-configuration
   - /docs/mobile-monitoring/new-relic-mobile-ios/install-configure/cocoapods-installation-configuration
-  - /docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation
 ---
 
-These procedures apply to iOS and tvOS apps using Cocoapods. For other types, see [iOS installation and configuration](/docs/mobile-monitoring-installation/ios-installation-and-configuration).
+We recommend that you use our [guided install](https://onenr.io/0qwLv87gkj5) to set up iOS monitoring. However, if you need to manually install for any reason, follow the steps below to install the New Relic iOS agent with Cocoapods.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:15]: for any reason

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx [hunk 12]
```diff
@@ -0,0 +1,117 @@
+---
+title: Swift Package Manager manual installation
+metaDescription: How to instrument to your iOS applications using Swift Package Manager.
+---
+
+We recommend that you use our [guided install](https://onenr.io/0qwLv87gkj5) to set up iOS monitoring. However, if you need to manually install for any reason, follow the steps below to install the New Relic iOS agent with Swift Package Manager.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx:6]: for any reason

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+
+## Install your iOS application [#installation]
+
+As part of the installation process, New Relic automatically generates an [application token](/docs/mobile-apps/viewing-your-application-token). This is a 40-character hexadecimal string for authenticating each mobile app you monitor in New Relic.
+
+To install and configure your iOS/tvOS application:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**.
+2. If applicable: From the Mobile Apps list, select **Add a new app**.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx:15]: Should Mobile Apps be bolded?

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+
+   ```
+   https://github.com/newrelic/newrelic-ios-agent-spm
+   ```
+
+   <Callout variant="tip">
+     If you receive an `artifact of binary target 'NewRelic' failed extraction: The operation couldn’t be completed. (TSCBasic.StringError error 1.)` error when extracting the package, please close Xcode, delete the Derrived Data folder, re-open Xcode, and try again.
+   </Callout>
+
+3. Select the NewRelic package product, select your target, and select Finish.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx:35]: Should Finish be bolded?

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+   ### **OPTIONAL**:
+    Add the following lines to your build script above the existing lines to skip symbol upload during debugging:
+    ```
+    if [ ${CONFIGURATION} = "Debug" ]; then
+        echo "Skipping DSYM upload CONFIGURATION: ${CONFIGURATION}"
+        exit 0
+    fi
+    ```
+
+6. If there is a checkbox below Run script that says "Run script: Based on Dependency analysis" please make sure it is not checked.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx:77]: Should Run script be formatted?

Comment (jeff-colucci) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx:77]: Also there should be a comma before please make sure

File: src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx [hunk 16]
```diff
@@ -49,7 +49,146 @@ If you haven't already, create your free New Relic account below to start monito
 
 To install the React Native agent, follow our [guided install](https://onenr.io/01wZvXb8Bw6), located directly in the UI.
 
-### Expo integration (optional) [#optional-expo-integration]
+If you need to manually install for any reason, follow these steps:
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx:52]: for any reason

File: src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx [hunk 17]
```diff
@@ -49,7 +49,146 @@ If you haven't already, create your free New Relic account below to start monito
...
+    // Optional:Enable or disable reporting data using different endpoints for US government clients
+    // fedRampEnabled: false
+  };
+NewRelic.startAgent(appToken,agentConfiguration);
+NewRelic.setJSAppVersion(version);
+AppRegistry.registerComponent(name, () => App);
+```
+
+<Callout variant="important">
+Please be sure to enter your proper application tokens. `appToken` is platform-specific. You  will need to generate the separate token for Android and iOS apps.
```
Comment (jeff-colucci) [inline src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx:115]: double space between "You will", it should also be "You'll" I think

**Review by jeff-colucci — DISMISSED** [review]
Just saw some small things, but I'm approving so you can merge once you take a look and see what changes you want to make. Overall, looks great!

File: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx (hunk 3 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx:17]:
```suggestion
Our New Relic Cordova agent monitors your Cordova mobile app and provides deep insights into your app's performance, errors, and user experience. The agent includes all New Relic mobile monitoring features we offer for native mobile apps. Once you install and configure the Cordova agent, you'll be able to:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx (hunk 4 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx:45]:
```suggestion
  1. Go to **[one.newrelic.com](https://onenr.io/02wdd6ngXwE) > Browser & Mobile > (choose Android or iOS)**, then complete the instrumentation steps in the UI. 
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx (hunk 5 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-cordova-phonegap/get-started-with-cordova-monitoring.mdx:67]:
```suggestion

To add optional configuration options, add the following configurations to the `--variable` argument, which will call them upon agent start:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx (hunk 9 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:41]:
```suggestion
  1. Go to **[one.newrelic.com](https://onenr.io/0EjOpynz0Q6) > Browser & Mobile > (choose Android or iOS)**, then complete the instrumentation steps in the UI. 
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx (hunk 10 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:142]:
```suggestion
4. Make sure your app requests `INTERNET` and `ACCESS_NETWORK_STATE` permissions by adding these lines to your `AndroidManifest.xml`:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx (hunk 13 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx:15]:
```suggestion
2. If applicable: From the **Mobile Apps** list, select **Add a new app**.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx (hunk 14 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx:35]:
```suggestion
3. Select the NewRelic package product, select your target, and select **Finish**.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx (hunk 15 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation.mdx:77]:
```suggestion
6. If there is a checkbox that says "Run script: Based on Dependency analysis" please make sure it is unchecked.
```

File: src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx (hunk 17 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx:115]:
```suggestion
Please be sure to enter your proper application tokens. `appToken` is platform-specific. You will need to generate the separate token for Android and iOS apps.
```

**Pull request merged**